seed/models/columns.py

```diff
@@ -543,6 +611,7 @@ class Column(models.Model):
     table_name = models.CharField(max_length=512, blank=True, db_index=True)
 
     display_name = models.CharField(max_length=512, blank=True)
+    column_description = models.TextField(max_length=1000, blank=True, default=None)
```

Comment:
FYI, max_length on text fields isn't enforced

Reply:
Thanks for the heads up. I did find this excerpt from [Django Fields documentation](https://docs.djangoproject.com/en/4.0/ref/models/fields/#textfield).

> If you specify a max_length attribute, it will be reflected in the [Textarea](https://docs.djangoproject.com/en/4.0/ref/forms/widgets/#django.forms.Textarea) widget of the auto-generated form field. However it is not enforced at the model or database level. Use a [CharField](https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.CharField) for that.

I don't think we are using the Textarea widget but wasn't sure if I should leave the max_length in there in case.